packages/react-scripts/template/README.md

```diff
@@ -1452,6 +1452,7 @@ If this doesn’t happen, try one of the following workarounds:
 * Some editors like Vim and IntelliJ have a “safe write” feature that currently breaks the watcher. You will need to disable it. Follow the instructions in [“Working with editors supporting safe write”](https://webpack.github.io/docs/webpack-dev-server.html#working-with-editors-ides-supporting-safe-write).
 * If your project path contains parentheses, try moving the project to a path without them. This is caused by a [Webpack watcher bug](https://github.com/webpack/watchpack/issues/42).
 * On Linux and macOS, you might need to [tweak system settings](https://webpack.github.io/docs/troubleshooting.html#not-enough-watchers) to allow more watchers.
+* Inside virtual machines such as (a Vagrant provisioned) VirtualBox, [use `CHOKIDAR_USEPOLLING=true npm start`](https://github.com/facebookincubator/create-react-app/issues/1049#issuecomment-261731734).
```

Comment:
Sorry, I didn't mean it that way.  Just meant that I'd prefer not to document a solution to a problem that might not exist (as it would clutter the docs with irrelevant info).  But it sounds like you're pretty sure it is a problem.  Have added the line to the docs.  :)